salt_stack/salt/build-lambda/files/makedomainenv

```diff
@@ -0,0 +1,118 @@
+#! /usr/bin/env bash
```

Comment:
Something went very wrong with the merge. This is the pre-Python 3.6 lambda file.

Reply:
Hm, I wonder if some of my initial work with getting downsample to work in lambda messed up the merge somehow.